lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1494,6 +1494,15 @@ static PrimDictConstructOp getDictConstructIfNotModified(Value torchDict) {
   return torchDict.getDefiningOp<Torch::PrimDictConstructOp>();
 }
 
+static PrimListConstructOp isListConstructNotModified(Value torchList) {
```

Comment:
Also, this function should be moved down to the `Aten__Contains__IntListOp` section, since it is not related to the `PrimTupleUnpackOp`